src/applications/vaos/tests/new-express-care-request/select-reason-page.unit.spec.jsx

```diff
@@ -35,10 +36,7 @@ describe('VAOS integration: Express Care form', () => {
     });
 
     await screen.findByText('Select a reason for your Express Care request');
-    fireEvent.click(screen.getByLabelText('Cough'));
-    await waitFor(
```

Comment:
I don't know if this will actually fix anything, but switched to making sure Continue works as a proxy for the radio button getting set (Continue won't call history if there's a validation error).